corehq/apps/users/models_sql.py

```diff
@@ -179,6 +184,41 @@ class RoleAssignableBy(models.Model):
     )
 
 
+class UserUpdateMeta(jsonobject.JsonObject):
+    updated_via = jsonobject.StringProperty()
+
+
+class UpdateDetails(jsonobject.JsonObject):
+    meta = jsonobject.ObjectProperty(UserUpdateMeta)
+    changes = jsonobject.DictProperty()
+
+
+class HQLogEntry(models.Model):
```

Comment:
This `models_sql` module is temporary for the duration of the couch-sql roles migration after which the models will be moved to the main `models` module. I think it would be better to put these directly in the `models` module.